wolfcrypt/test/test.c

```diff
@@ -6392,8 +6400,13 @@ int ecc_test(void)
 
 #ifdef HAVE_ECC_SIGN
     /* test DSA sign hash */
-    for (i = 0; i < (int)sizeof(digest); i++)
+    for (i = 0; i < (int)sizeof(digest); i++) {
+    #ifdef ECC_TEST_VERIFY_ZERO
```

Comment:
That is correct. With the way I set it up the digest will be all 0's for FP_ECC. You are right, ideally FP_ECC would test both (all zero's and the sequence of digits). In fact we might be able to have wc_ecc_verify_hash test both scenarios all the time.